avalon/nuke/pipeline.py

```diff
@@ -1,43 +1,130 @@
-"""Pipeline functionality specifically for Nuke
+import os
+import importlib
+from .. import api, io
+import contextlib
+from pyblish import api as pyblish
+from ..vendor import toml
+import nuke
 
-Hi Arvid!
```

Comment:
Haha, remnants from the old days. :) Arvid was the Nuke compositor at Mindbender where/when Avalon was born, and this file was made as a template for him to fill in.

Reply:
:) will remove the trace.